lib/sqewer/connection.rb

```diff
@@ -39,12 +39,26 @@ def self.default
     raise "SQS_QUEUE_URL not set in the environment. This is the queue URL Sqewer uses by default."
   end
 
+  # Returns a singleton of Aws::SQS::Client
+  def self.client
+    # It's better using a singleton client to prevent making a lot of HTTP
+    # requests to the AWS metadata endpoint when getting credentials.
+    # Maybe in the future, we can remove @client and use Storm.client only.
```

Comment:
We should remove this reference, given sqewer is a public repo 😊

Reply:
ow, this is not necessary indeed, tks!